src/app/LoginController.java

```diff
@@ -41,6 +46,10 @@
 
    private Timer timer;
 
+    public LoginController() throws AccountDaoException {
+        this.loginDao = AccountDaoFactory.getDao();
```

Comment:
Should probably go in login() so it's dereferenced when login finishes. We don't need to refer to it elsewhere (unlike timer).